webrtc.html

```diff
@@ -8099,7 +8101,8 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
                 </li>
                 <li>
                   <p>If the <code><a data-for="RTCDataChannel">id</a></code>
-                  attribute is uninitialized (not set via the dictionary),
+                  attribute is <code>null</code> (due to no ID being
+                  passed into <code>createDataChannel</code>),
                   and the DTLS role of the SCTP transport has already been
                   negotiated, then initialize <code><a data-for=
                   "RTCDataChannel">id</a></code> to a value generated by the
```

Comment:
I'm not seeing this. Here's what I intended, in pseudo-code form:

```
dataChannel.id = init.id;
if (!dataChannel.id) {
  if (dtls_role_negotiated) {
    try {
      dataChannel.id = PickAvailableId();
    } catch (e) {
      throw ResourceInUse;
    }
  }
}

So it will end up null if no ID was provided and the DTLS role is not yet negotiated.